data/pathfinder/paizo/roleplaying_game/advanced_race_guide/arg_abilitycategories.lst

```diff
@@ -155,7 +157,6 @@ ABILITYCATEGORY:Weapon Familiarity					VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO
 ABILITYCATEGORY:Hatred							VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO							CATEGORY:Internal			TYPE:Hatred																									DISPLAYLOCATION:Race Builder Design
 ABILITYCATEGORY:Race Builder Skill Bonus				VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO							CATEGORY:Internal			TYPE:Race Builder Skill Bonus																						DISPLAYLOCATION:Race Builder Design
 
-ABILITYCATEGORY:Race Builder Energy Resistance				VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO							CATEGORY:Internal			TYPE:BuilderEnergyResistanceChoice																						DISPLAYLOCATION:Race Builder Design
```

Comment:
Looking at the copy in GITHUB, it's in there at line 160. I had to delete two PR's because I was having trouble rebasing that day. The rebasing issue kept leaving this line out. My 3rd PR included that line. It's in there!! 

    On Tuesday, June 14, 2016 4:26 PM, Andrew Maitland <notifications@github.com> wrote:
 

 In data/pathfinder/paizo/roleplaying_game/advanced_race_guide/arg_abilitycategories.lst:> @@ -155,7 +157,6 @@ ABILITYCATEGORY:Weapon Familiarity					VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO
>  ABILITYCATEGORY:Hatred							VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO							CATEGORY:Internal			TYPE:Hatred																									DISPLAYLOCATION:Race Builder Design
>  ABILITYCATEGORY:Race Builder Skill Bonus				VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO							CATEGORY:Internal			TYPE:Race Builder Skill Bonus																						DISPLAYLOCATION:Race Builder Design
>  
> -ABILITYCATEGORY:Race Builder Energy Resistance				VISIBLE:QUALIFY	EDITABLE:YES	EDITPOOL:NO							CATEGORY:Internal			TYPE:BuilderEnergyResistanceChoice																						DISPLAYLOCATION:Race Builder Design
This needs to be restored.—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub, or mute the thread.